hs_core/views/utils.py

```diff
@@ -329,6 +329,7 @@ def run_ssh_command(host, uname, exec_cmd, pwd=None, private_key_file=None):
     return output
 
 
+# TODO: can we remove this?
```

Comment:
unsure what HYRAX is...I'm wondering if we can get rid of this

Reply:
hyrax == thredds
tds-proxy.hydroshare.org

Reply:
Thank you!